src/game.py

```diff
@@ -1,2 +1,20 @@
+# Start the game
```

Comment:
Them comments aren't really useful, the code IS the documentation for such small functions